lib/compress/zstd_compress_internal.h

```diff
@@ -1264,6 +1268,31 @@ MEM_STATIC void ZSTD_debugTable(const U32* table, U32 max)
 
 #endif
 
+/* Short Cache */
+
+typedef enum {
+    /* TODO: this is a good place to document short cache */
```

Comment:
Yeah, this enum is generic and not really related to short cache except, but right after it would be a good place.